pandas/tests/indexing/test_floats.py

```diff
@@ -94,11 +94,11 @@ def test_scalar_non_numeric(self, index_func):
                 else:
                     error = TypeError
                     msg = (
-                        r"cannot do (label|positional) indexing "
-                        r"on {klass} with these indexers \[3\.0\] of "
-                        r"type float|"
-                        "Cannot index by location index with a "
-                        "non-integer key".format(klass=type(i).__name__)
+                        fr"cannot do (label|positional) indexing "
```

Comment:
Same comment as @MomIsBestFriend no need to add the `f` prefix where no replacement is being made. Applicable in quite a few spots in the PR

Reply:
The reason I included them is because those are multiline strings, and basically have the `f` or `fr` identifier does not harm and improves clarity. If my point is not valid I can remove them